src/clj/game/core/commands.clj

```diff
@@ -264,22 +269,22 @@
         nil)
       (case command
         "/adv-counter" #(command-adv-counter %1 %2 value)
-        "/bp"         #(swap! %1 assoc-in [%2 :bad-publicity :base] (max 0 value))
+        "/bp"         #(swap! %1 assoc-in [%2 :bad-publicity :base] (constrain-value value -1000 1000))
```

Comment:
Why can bad pub be negative?

Reply:
We aren't real consistent about what value gets modified when you use these commands. I figure if someone is stuck with a bp modifier, you can make the base value negative to correct for it. I think I do something similar in a few other cases as well.